.github/file-paths.yaml

```diff
@@ -80,7 +80,6 @@ e2e_specs: &e2e_specs
 e2e_all:
   - *default
   - *ci
-  - *e2e_specs
```

Comment:
The gist of this change is to decouple pure E2E spec and helper changes from all other (source code) changes that will trigger the actual build process.